test/e2e/tests/single_zone_e2e_test.go

```diff
@@ -287,6 +287,36 @@ var _ = Describe("GCE PD CSI Driver", func() {
 
 	})
 
+	It("Should complete entire disk lifecycle with underspecified volume ID and missing volume", func() {
```

Comment:
Nit: should we also add this test case for pd-extreme?

Reply:
Done.